debian/enterprise-replicator/Dockerfile

```diff
@@ -15,7 +15,7 @@
 
 # Builds a docker image for Replicator
 
-FROM confluentinc/cp-kafka-connect-base
+FROM confluentinc/cp-server-connect-base
```

Comment:
do we need to have both versions for replicator, one inheriting from cp-kafka and another from cp-server?

Reply:
Nope, Replicator is already an enterprise product so there's no need to have an additional image that inherits from our non-enterprise image.